src/jit-diff/jit-diff.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <DebugType>portable</DebugType>
+    <AssemblyName>jit-diff</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackageId>jit-diff</PackageId>
+    <PackageTargetFallback>$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.1.1" />
+    <PackageReference Include="Microsoft.DotNet.Cli.Utils" Version="1.0.0-rc4-*" />
+    <PackageReference Include="System.CommandLine" Version="0.1.0-*" />
```

Comment:
Yep, thanks, I'd done that with Microsoft.DotNet.Cli.Utils but missed System.CommandLine.  I've updated to make all references consistent across all projects.